executor/infoschema_cluster_table_test.go

```diff
@@ -326,3 +326,37 @@ func TestTableStorageStats(t *testing.T) {
 
 	tk.MustQuery("select count(1) from information_schema.TABLE_STORAGE_STATS where TABLE_SCHEMA = 'mysql'").Check(testkit.Rows(strconv.Itoa(result)))
 }
+
+func TestIssue42619(t *testing.T) {
+	s := createInfosSchemaClusterTableSuite(t)
+	mockAddr := s.mockAddr
+	store := &mockStore{
+		s.store.(helper.Storage),
+		mockAddr,
+	}
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+
+	tk.MustExec("create table t (a int, b int, index idx(a))")
+	tk.MustQuery("select * from information_schema.TABLE_STORAGE_STATS where TABLE_SCHEMA = 'test' and TABLE_NAME='t'").Check(
+		testkit.Rows("test t 88 1 1 1 1 1"))
```

Comment:
If you add another system table, the tableID will no longer be 88. This unit test often needs to be modified. Maybe not check tableID is better.

Reply:
I removed the table IDs. PTAL~